core/templates/dev/head/pages/collection_editor/editor_tab/CollectionLinearizerService.js

```diff
@@ -58,11 +58,6 @@ oppia.factory('CollectionLinearizerService', [
       return nodeIndex > 0 ? nodeIndex - 1 : null;
```

Comment:
do you think it's worth inlining this too (for parity with the now-removed getNodeIndexRightOf)? If it's used in only one place then it might be worth it, otherwise devs will wonder why you have one and not the other.

(Or the reverse is fine too -- reinstating the other function and using it. Just commenting on the asymmetry.)